Wason’s Selection Task.md

```diff
@@ -0,0 +1,107 @@
+2. The subjects were told that cards with letters on their front had numbers on their back and vice uersa. 
+3. In the selection task the experimenter pointed to each card in turn and asked the subject whether knowing what was on the other side would enable him to find out whether the sentence was true or false. 
+4. In the experimental group, the experimenter pointed to each card and asked the subject to name a letter (or number) on the other side which would make the sentence false, or to say “none,” if none would do so.
+
+*Experiment 2 - The restricted contingency programme*
+
+The subjects were allocated alternately to the groups and tested individually. Six subjects in the experimental group were tested with sentence (1) and seven
+with sentence (2), these proportions being reversed in the control group. 
+
+1. Before presenting the conditional sentence the subjects in both groups examined the cards briefly to familiarize them with the fact that there was always a geometric shape on one side and a coloured scribble on the other side.
+2. In the experimental group the conditional sentence was presented and the four programme cards were handed to the subjects who were asked to pick out “the one card which makes the rule false” (i.e. falsifies the conditional sentence). 
+3. They were then asked to pick out any which “prove the rule true.” It was explained to them that their decisions meant that the converse of the sentence could not be assumed - “that the rule only held one way.” 
+4. The subjects in the control group were given a similar amount of time to understand the conditional sentence without any explanation of its meaning.
+
+The selection task had three phases which were the same for both groups.
+
+**Phase I.** The eight selection task cards were placed on the desk front side up in a random array, and the subject was asked to pick out “all those cards, but only those cards, which would show you, if you knew what was on the other side, that the rule was true or false.”
+
+**Phase 2.** The subject was asked to turn over all those cards which he had selected and evaluate the conditional sentence with respect to each: “tell me whether each proves the rule true or proves it false.”
+
+**Phase 3.** The subject was invited to project falsity on to the residual cards i.e. those cards which had not been selected. Each pair of cards (representing the same value) was pointed to in turn, starting with the Q' cards, if these had not been selected: “could anything on the back of those cards make the rule false?”
+
+
+## Roles 
+
+No specific roles for both experiments. All of the participants were asked to complete the selection task following the instructions and steps.
+
+## Instructions
+
+*Experiment 1 - The projection of falsity*
+
+(pending)
+
+*Experiment 2 - The restricted contingency programme*
+
+(pending)
+Also, refer to the three phases mentioned in Steps.
+
+# Criteria
+## Performance calculation
```

Comment:
Maybe in this section it would be more convenient to just rephrase performance calculation criteria. Something like "In Experiment 1, participants were considered to... if they chose Q...